the-isme-journal.csl

```diff
@@ -5,6 +5,7 @@
     <id>http://www.zotero.org/styles/the-isme-journal</id>
     <link href="http://www.zotero.org/styles/the-isme-journal" rel="self"/>
     <link href="http://mts-isme.nature.com/cgi-bin/main.plex?form_type=display_auth_instructions" rel="documentation"/>
+    <link href="http://www.nature.com/ismej/ismej_new_gta.pdf" rel="documentation"/>
```

Comment:
If this documentation link supersedes the old one, it's better to just replace the original instead of adding a second one.